calendar_v17/GoogleMapsScraper.py

```
@@ -0,0 +1,1510 @@
+                search_box.send_keys(query)
+                search_box.send_keys(Keys.RETURN)
+                time.sleep(delay + 2)  # Wait for results to load
+                return True
+            except Exception as e:
+                if attempt < max_attempts - 1:
+                    self.queue.put(('status', f"Search attempt {attempt+1} failed, retrying..."))
+                    time.sleep(delay)
+                    try:
+                        driver.refresh()
+                        time.sleep(delay)
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)